whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/processor/UpdateNotificationFileProcessor.java

```diff
@@ -45,10 +48,12 @@ public class UpdateNotificationFileProcessor {
 
     public UpdateNotificationFileProcessor(final NrtmRestClient nrtmRestClient,
                                            final Nrtm4ClientInfoRepository nrtm4ClientMirrorDao,
-                                           final SnapshotImporter snapshotImporter) {
+                                           final SnapshotImporter snapshotImporter,
+                                           final DeltaImporter deltaImporter) {
         this.nrtmRestClient = nrtmRestClient;
         this.nrtm4ClientMirrorDao = nrtm4ClientMirrorDao;
         this.snapshotImporter = snapshotImporter;
+        this.deltaImporter = deltaImporter;
     }
 
     @Transactional(rollbackFor = Exception.class)
```

Comment:
You need to specify transaction manager ???

Reply:
I need to fix this. It is not working

Reply:
talked over zoom, created a task for fixing it